index.html

```diff
@@ -16,45 +16,44 @@
-                   <img src="./assets/Images/logo/Standard Logo Files/Original.png" id="logo" alt="logo">
-                </div>                   
+                    <img src="./assets/Images/logo/Standard Logo Files/Original.png" id="logo" alt="logo">
+                </div>
             </div>
             <div class="column">
-               <h1 class="title has-text-grey-dark has-text-centered">
-                 30 Minute Workout!
-               </h1>
-
-               <h2 class="subtitle has-text-grey-dark has-text-centered">
-                  Survival of the Fittest
-               </h2>
-           </div>
-                 Survival of the Fittest
-             </h2>
-         </div>
+                <h1 class="title has-text-grey-dark has-text-centered">
+                    30 Minute Workout!
+                </h1>
+
+                <h2 class="subtitle has-text-grey-dark has-text-centered">
+                    Survival of the Fittest
+                </h2>
+            </div>
+            Survival of the Fittest
+            </h2>
+        </div>
     </section>
     <section id="links">
-      
+
         <!-- Code for the Navbar -->
         <nav class="navbar is-primary" role="navigate" aria-label="main-navigation">
-                    <div class="navbar-brand">
-                        <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false"
-                            data-target="navBasic">
-                            <span aria-hidden="true"></span>
-                            <span aria-hidden="true"></span>
-                            <span aria-hidden="true"></span>
-
-                        </a>
-                    </div>
-                    <div id="navbarBasic" class="navbar-menu is active">
-                        <div class="navbar-start">
-                                <a href="nutrition.html" class="navbar-item has-text-grey-darker" id="nutrition_link">Meal Plan</a>
-                                <a href="" class="navbar-item has-text-grey-dark" id="challenge_link">Challenge a Friend</a>
-                        </div>
-                    </div>
-
-                </nav>
-            </section>
-        <section class="container is fluid" id="profile">
+            <div class="navbar-brand">
+                <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navBasic">
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+
+                </a>
+            </div>
+            <div id="navbarBasic" class="navbar-menu is active">
+                <div class="navbar-start">
+                    <a href="#" class="navbar-item has-text-grey-darker" id="nutrition_link">Meal Plan</a>
+                    <a href="#" class="navbar-item has-text-grey-dark" id="challenge_link">Challenge a Friend</a>
```

Comment:
Let's just remove links that go nowhere or have them do something.

Reply:
Corrected links